### DISCUSSION TITLE
Helm Secrets with Argocd

### DISCUSSION THREAD

**Discussion by KeerthigaKrishnan** [argoproj/argo-cd#5486] [General]
Hi Team, 

Do we have support for multiple keys while bringing up the argocd repo-server?. I have created key in kms, is it possible to use multiple key in argocd-repo server ?.

**Comment by jannfis** [comment]
Hi @KeerthigaKrishnan,

Argo CD is not opinionated about the secrets management method you will use, so their is no official support for things like Helm secrets. Also, we do not provide any mechanisms for handling secret keys, because that would be out of scope for Argo CD. If you want to use Helm Secrets, you will have to provide secret keys on your own, e.g. by mounting a `Secret` to the repo-server pods. I'm sure some people already have done so, and might also have written some how-tos for that.
